packages/customize-uploader/bin/cli.js

```diff
@@ -25,7 +25,7 @@ const cli = meow(
     $ kintone-customize-uploader <manifestFile>
   Options
     --base-url Base-url of your kintone
-    --domain Domain of your kintone (If you set --base-url, this value is not necessary.)
+    --domain Domain of your kintone (This value is deprecated. Please you use base-url.)
```

Comment:
```suggestion
    --domain Domain of your kintone (This value is deprecated. Please use --base-url.)
```

"you" seems unnecessary, and if we follow `--oauth-token`'s description, I prefer appending `--` prefix 👍